src/components/EmptyState/EmptyState.js

```diff
@@ -2,16 +2,49 @@ import './EmptyState.css'
 import React from 'react'
 
 function EmptyState() {
+  /* <img src='img/squirrel_illustration_empty.png' /> */
   return (
     <div className='empty_state_wrapper'>
       <div className='empty_state'>
-        <img src='img/squirrel_illustration_empty.png' />
-        <h1>welcome to acorn</h1>
-        <h4>
+        <div class='outer'>
+          <div class='middle'>
+            <div class='main clearfix'>
```

Comment:
can you rename the 'main' class here and in the css. It's too generic. It doesn't matter what you call it, just make it something less universal, and more specific

Reply:
ok